StringR.Store/Model/Order.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Order.swift
+//  StringR.Store
+//
+//  Created by Marcus Christiansen on 13/09/2019.
+//  Copyright © 2019 StringR. All rights reserved.
+//
+
+import Foundation
+
+struct Order {
+
+    var orderId: String
+    var customerId: String
+    var stringerId: String
+    var racketType: Racket.RacketType
+    var tensionVertical: Double
+    var tensionHorizontal: Double
+    var stringId: String
+    var deliveryDate: Double
```

Comment:
```suggestion
    var deliveryDate: Long
```

Should this not be a Long? And same with the ones below 😀